packages/node_modules/@ciscospark/widget-space/src/container.js

```diff
@@ -193,7 +195,30 @@ export class SpaceWidget extends Component {
     } = props;
 
     const {formatMessage} = props.intl;
-    if (sparkInstance && spaceDetails) {
+    if (widgetStatus.get(`isError`)) {
+      const error = widgetSpace.get(`error`);
+      const errorTitle = formatMessage(messages.unableToLoad);
+      let errorSubtitle;
+      switch (error.name) {
+      case `Bad Space Id`:
+        errorSubtitle = formatMessage(messages.badSpaceId);
+        break;
+      case `NotFound`:
+        errorSubtitle = formatMessage(messages.errorNotFound);
+        break;
+      default:
+        errorSubtitle = formatMessage(messages.unknownError);
```

Comment:
Gotcha, yea, we're working on getting Sentry in place somewhere so that we can start capturing exceptions